app/modules/platform/actions/apiSaga/index.js

```diff
@@ -2,15 +2,17 @@
 
 import apiDeleteToken from './apiDeleteToken';
 import apiPostConfirmation from './apiPostConfirmation';
-import apiPostEmailToConfirmation from './apiPostEmailToConfirmation';
-import apiPostEmailToPassword from './apiPostEmailToPassword';
+import apiPatchConfirmation from './apiPatchConfirmation';
+import apiPostPassword from './apiPostPassword';
+import apiPatchPassword from './apiPatchPassword';
 import apiPostSigninToTokenAndGetUserAuth from './apiPostSigninToTokenAndGetUserAuth';
 
 const apiSagaActions = {
   apiDeleteToken,
   apiPostConfirmation,
-  apiPostEmailToConfirmation,
-  apiPostEmailToPassword,
+  apiPatchConfirmation,
+  apiPostPassword,
+  apiPatchPassword,
   apiPostSigninToTokenAndGetUserAuth,
 };
```

Comment:
Can we add an ESLint rule for this? Something like this:

```

    {
      'files': [
        'index.js',
      ],
      'rules': {
        'sort-keys': ['error', 'asc', { 'caseSensitive': true, 'natural': false }],
      }
    }
```

But this one reorders every object in index files...

Reply:
Like that it would affect all files named `index`, which includes most components. We could specify a few file/folder name patterns for index files used in modules, though. I'll give it a shot :)